src/Peak_Tracking.py

```diff
@@ -1,39 +1,43 @@
-'''
-My objective is to generate a new 2D matrix (called "tracked") with shape (a,b) where:
-    a: total measurements/gr files
-    b: max number of peaks from all measurements 
-The matrix consists of the peak positions, and the "same" peaks (across rows) populate the same index (column) in the matrix.
-The value 'NaN' indicates undetected peak in the measurement. 
-
-This version of code has only considered the first two (dwell) files.
-I am still in progress of making the algorithm. 
-'''
 import numpy as np
 
-file = np.load("../peak_positions_npz/pdf_dwell_peaks.npz")
-keys = list(file.keys())
-print(keys)
+def track_peaks(npz_file):
+    """
+    Generate a 2D matrix (called "tracked") with shape (a, b) such that:
+        a: total measurements/gr files
+        b: max number of peaks from all measurements 
+    The matrix consists of the peak positions, and the "same" peaks (across rows) populate the same index (column) in the matrix.
+    The value 'NaN' indicates an undetected peak in the measurement. 
+
+    This version of code has only considered the first two (dwell) files.
+    I am still in progress of making the algorithm.
+    Args:
+        npz_file: .npz file from which data will be loaded/read.
+    Returns: 2D matrix containing tracked peaks.
+    """
 
-i = 0
+    data_from_npz = np.load(npz_file)
```

Comment:
Variable name "data_from_npz" is definitely more descriptive than just "file". Will change this.